cluster/ci/config/prow/acsengine/plugins.yaml

```diff
@@ -0,0 +1,29 @@
+approve:
+- repos:
+  - kargakis/acs-engine
+
+config_updater:
+  maps:
+    cluster/ci/config/prow/acsengine/config.yaml:
+      name: config
+    cluster/ci/config/prow/acsengine/plugins.yaml:
+      name: plugins
+
+plugins:
+  kargakis/acs-engine:
+  - approve
+  - assign
+  - cat
+  - dog
+  - help
+  - hold
+  - label
+  - lgtm
+  - lifecycle
+  - shrug
+  - size
+  - wip
+  - yuks
+
+  openshift/release:
```

Comment:
Depending on where all these config files are stored, this needs to be updated accordingly.

Reply:
@kargakis does this mean I need to add the config files to the acs-engine repo?

Reply:
You can store the config related to the deployment of prow (deployments, ingress, services) wherever you like.

The prow config file is more important and what most users are going to interface with. You should be able to store both config.yaml and plugins.yaml in acs-engine, so you likely want something like this:
```yaml
approve:
- repos:
  - Azure/acs-engine

config_updater:
  maps:
    .prowci/config.yaml:
      name: config
    .prowci/plugins.yaml:
      name: plugins

plugins:
  Azure/acs-engine:
  - approve
  - assign
  - cat
  - config-updater
  - dog
  - help
  - hold
  - label
  - lgtm
  - lifecycle
  - shrug
  - size
  - wip
  - yuks
```
We can walk through this and our own setup in our call next week.